src/components/PsSideBar/PsSideBar.js

```diff
@@ -0,0 +1,69 @@
+import { Collapse, Layout } from 'antd';
+import { useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import SortingList from '../SortingList/SortingList';
+import FiltersList from '../FiltersList/FiltersList';
+
+import {
+  fetchSortingOption,
+  setActiveSortingOption,
+} from '../../redux/sortingOptionSlice';
+import { fetchFilters, setActiveFiters } from '../../redux/filtersListSlice';
+
+const { Panel } = Collapse;
+const { Sider: AntSider } = Layout;
+
+export default function PsSideBar() {
+  const sortingList = useSelector((state) => state.sorting.sortingOptionsList);
+  const activeSortingOption = useSelector(
+    (state) => state.sorting.activeSortingOption
+  );
+  const filtersList = useSelector((state) => state.filters.filtersList);
+  const activeFilters = useSelector((state) => state.filters.activeFilters);
+
+  const dispatch = useDispatch();
+
+  const onSortingOptionChange = useCallback(
+    (value) => {
+      dispatch(setActiveSortingOption(value));
+    },
+    [dispatch]
+  );
+
+  const onActiveFitersChange = useCallback(
+    (value) => {
+      dispatch(setActiveFiters(value));
+    },
+    [dispatch]
+  );
+
+  useEffect(() => {
+    dispatch(fetchFilters());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchSortingOption());
```

Comment:
Unnecessary duplication of useEffect. As they have the same dependency array you can do both dispatches in the one hook

Reply:
fixed